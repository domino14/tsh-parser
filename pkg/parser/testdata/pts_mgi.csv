posrange,MGILeague-Div1,Masters,Open,MGILeague-Div2,Intermediate
1,6000,5000,4500,3000,3000
2,5000,4000,3500,2500,2500
3,4250,3500,3000,2000,2000
4,3500,3000,2500,1750,1750
5,3000,2750,2250,1500,1500
6-10,2750,2500,2000,1250,1250
11-15,2500,2250,1800,1050,1050
16-20,2250,2000,1600,900,900
21-25,2000,1750,1400,800,800
26-30,1750,1500,1200,700,700
31-35,1500,1250,1000,600,600
36-40,1250,1000,800,500,500
41-45,1000,750,600,400,400
46-50,750,500,500,350,350
51-55,500,100,400,300,300
56-60,250,100,300,250,250
61-70,100,100,250,100,100
71-80,100,100,200,100,100
81-90,100,100,150,100,100
91-100,100,100,100,100,100