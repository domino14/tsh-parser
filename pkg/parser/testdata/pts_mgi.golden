100:Intermediate: 100
100:MGILeague-Div1: 100
100:MGILeague-Div2: 100
100:Masters: 100
100:Open: 100
10:Intermediate: 1250
10:MGILeague-Div1: 2750
10:MGILeague-Div2: 1250
10:Masters: 2500
10:Open: 2000
11:Intermediate: 1050
11:MGILeague-Div1: 2500
11:MGILeague-Div2: 1050
11:Masters: 2250
11:Open: 1800
12:Intermediate: 1050
12:MGILeague-Div1: 2500
12:MGILeague-Div2: 1050
12:Masters: 2250
12:Open: 1800
13:Intermediate: 1050
13:MGILeague-Div1: 2500
13:MGILeague-Div2: 1050
13:Masters: 2250
13:Open: 1800
14:Intermediate: 1050
14:MGILeague-Div1: 2500
14:MGILeague-Div2: 1050
14:Masters: 2250
14:Open: 1800
15:Intermediate: 1050
15:MGILeague-Div1: 2500
15:MGILeague-Div2: 1050
15:Masters: 2250
15:Open: 1800
16:Intermediate: 900
16:MGILeague-Div1: 2250
16:MGILeague-Div2: 900
16:Masters: 2000
16:Open: 1600
17:Intermediate: 900
17:MGILeague-Div1: 2250
17:MGILeague-Div2: 900
17:Masters: 2000
17:Open: 1600
18:Intermediate: 900
18:MGILeague-Div1: 2250
18:MGILeague-Div2: 900
18:Masters: 2000
18:Open: 1600
19:Intermediate: 900
19:MGILeague-Div1: 2250
19:MGILeague-Div2: 900
19:Masters: 2000
19:Open: 1600
1:Intermediate: 3000
1:MGILeague-Div1: 6000
1:MGILeague-Div2: 3000
1:Masters: 5000
1:Open: 4500
20:Intermediate: 900
20:MGILeague-Div1: 2250
20:MGILeague-Div2: 900
20:Masters: 2000
20:Open: 1600
21:Intermediate: 800
21:MGILeague-Div1: 2000
21:MGILeague-Div2: 800
21:Masters: 1750
21:Open: 1400
22:Intermediate: 800
22:MGILeague-Div1: 2000
22:MGILeague-Div2: 800
22:Masters: 1750
22:Open: 1400
23:Intermediate: 800
23:MGILeague-Div1: 2000
23:MGILeague-Div2: 800
23:Masters: 1750
23:Open: 1400
24:Intermediate: 800
24:MGILeague-Div1: 2000
24:MGILeague-Div2: 800
24:Masters: 1750
24:Open: 1400
25:Intermediate: 800
25:MGILeague-Div1: 2000
25:MGILeague-Div2: 800
25:Masters: 1750
25:Open: 1400
26:Intermediate: 700
26:MGILeague-Div1: 1750
26:MGILeague-Div2: 700
26:Masters: 1500
26:Open: 1200
27:Intermediate: 700
27:MGILeague-Div1: 1750
27:MGILeague-Div2: 700
27:Masters: 1500
27:Open: 1200
28:Intermediate: 700
28:MGILeague-Div1: 1750
28:MGILeague-Div2: 700
28:Masters: 1500
28:Open: 1200
29:Intermediate: 700
29:MGILeague-Div1: 1750
29:MGILeague-Div2: 700
29:Masters: 1500
29:Open: 1200
2:Intermediate: 2500
2:MGILeague-Div1: 5000
2:MGILeague-Div2: 2500
2:Masters: 4000
2:Open: 3500
30:Intermediate: 700
30:MGILeague-Div1: 1750
30:MGILeague-Div2: 700
30:Masters: 1500
30:Open: 1200
31:Intermediate: 600
31:MGILeague-Div1: 1500
31:MGILeague-Div2: 600
31:Masters: 1250
31:Open: 1000
32:Intermediate: 600
32:MGILeague-Div1: 1500
32:MGILeague-Div2: 600
32:Masters: 1250
32:Open: 1000
33:Intermediate: 600
33:MGILeague-Div1: 1500
33:MGILeague-Div2: 600
33:Masters: 1250
33:Open: 1000
34:Intermediate: 600
34:MGILeague-Div1: 1500
34:MGILeague-Div2: 600
34:Masters: 1250
34:Open: 1000
35:Intermediate: 600
35:MGILeague-Div1: 1500
35:MGILeague-Div2: 600
35:Masters: 1250
35:Open: 1000
36:Intermediate: 500
36:MGILeague-Div1: 1250
36:MGILeague-Div2: 500
36:Masters: 1000
36:Open: 800
37:Intermediate: 500
37:MGILeague-Div1: 1250
37:MGILeague-Div2: 500
37:Masters: 1000
37:Open: 800
38:Intermediate: 500
38:MGILeague-Div1: 1250
38:MGILeague-Div2: 500
38:Masters: 1000
38:Open: 800
39:Intermediate: 500
39:MGILeague-Div1: 1250
39:MGILeague-Div2: 500
39:Masters: 1000
39:Open: 800
3:Intermediate: 2000
3:MGILeague-Div1: 4250
3:MGILeague-Div2: 2000
3:Masters: 3500
3:Open: 3000
40:Intermediate: 500
40:MGILeague-Div1: 1250
40:MGILeague-Div2: 500
40:Masters: 1000
40:Open: 800
41:Intermediate: 400
41:MGILeague-Div1: 1000
41:MGILeague-Div2: 400
41:Masters: 750
41:Open: 600
42:Intermediate: 400
42:MGILeague-Div1: 1000
42:MGILeague-Div2: 400
42:Masters: 750
42:Open: 600
43:Intermediate: 400
43:MGILeague-Div1: 1000
43:MGILeague-Div2: 400
43:Masters: 750
43:Open: 600
44:Intermediate: 400
44:MGILeague-Div1: 1000
44:MGILeague-Div2: 400
44:Masters: 750
44:Open: 600
45:Intermediate: 400
45:MGILeague-Div1: 1000
45:MGILeague-Div2: 400
45:Masters: 750
45:Open: 600
46:Intermediate: 350
46:MGILeague-Div1: 750
46:MGILeague-Div2: 350
46:Masters: 500
46:Open: 500
47:Intermediate: 350
47:MGILeague-Div1: 750
47:MGILeague-Div2: 350
47:Masters: 500
47:Open: 500
48:Intermediate: 350
48:MGILeague-Div1: 750
48:MGILeague-Div2: 350
48:Masters: 500
48:Open: 500
49:Intermediate: 350
49:MGILeague-Div1: 750
49:MGILeague-Div2: 350
49:Masters: 500
49:Open: 500
4:Intermediate: 1750
4:MGILeague-Div1: 3500
4:MGILeague-Div2: 1750
4:Masters: 3000
4:Open: 2500
50:Intermediate: 350
50:MGILeague-Div1: 750
50:MGILeague-Div2: 350
50:Masters: 500
50:Open: 500
51:Intermediate: 300
51:MGILeague-Div1: 500
51:MGILeague-Div2: 300
51:Masters: 100
51:Open: 400
52:Intermediate: 300
52:MGILeague-Div1: 500
52:MGILeague-Div2: 300
52:Masters: 100
52:Open: 400
53:Intermediate: 300
53:MGILeague-Div1: 500
53:MGILeague-Div2: 300
53:Masters: 100
53:Open: 400
54:Intermediate: 300
54:MGILeague-Div1: 500
54:MGILeague-Div2: 300
54:Masters: 100
54:Open: 400
55:Intermediate: 300
55:MGILeague-Div1: 500
55:MGILeague-Div2: 300
55:Masters: 100
55:Open: 400
56:Intermediate: 250
56:MGILeague-Div1: 250
56:MGILeague-Div2: 250
56:Masters: 100
56:Open: 300
57:Intermediate: 250
57:MGILeague-Div1: 250
57:MGILeague-Div2: 250
57:Masters: 100
57:Open: 300
58:Intermediate: 250
58:MGILeague-Div1: 250
58:MGILeague-Div2: 250
58:Masters: 100
58:Open: 300
59:Intermediate: 250
59:MGILeague-Div1: 250
59:MGILeague-Div2: 250
59:Masters: 100
59:Open: 300
5:Intermediate: 1500
5:MGILeague-Div1: 3000
5:MGILeague-Div2: 1500
5:Masters: 2750
5:Open: 2250
60:Intermediate: 250
60:MGILeague-Div1: 250
60:MGILeague-Div2: 250
60:Masters: 100
60:Open: 300
61:Intermediate: 100
61:MGILeague-Div1: 100
61:MGILeague-Div2: 100
61:Masters: 100
61:Open: 250
62:Intermediate: 100
62:MGILeague-Div1: 100
62:MGILeague-Div2: 100
62:Masters: 100
62:Open: 250
63:Intermediate: 100
63:MGILeague-Div1: 100
63:MGILeague-Div2: 100
63:Masters: 100
63:Open: 250
64:Intermediate: 100
64:MGILeague-Div1: 100
64:MGILeague-Div2: 100
64:Masters: 100
64:Open: 250
65:Intermediate: 100
65:MGILeague-Div1: 100
65:MGILeague-Div2: 100
65:Masters: 100
65:Open: 250
66:Intermediate: 100
66:MGILeague-Div1: 100
66:MGILeague-Div2: 100
66:Masters: 100
66:Open: 250
67:Intermediate: 100
67:MGILeague-Div1: 100
67:MGILeague-Div2: 100
67:Masters: 100
67:Open: 250
68:Intermediate: 100
68:MGILeague-Div1: 100
68:MGILeague-Div2: 100
68:Masters: 100
68:Open: 250
69:Intermediate: 100
69:MGILeague-Div1: 100
69:MGILeague-Div2: 100
69:Masters: 100
69:Open: 250
6:Intermediate: 1250
6:MGILeague-Div1: 2750
6:MGILeague-Div2: 1250
6:Masters: 2500
6:Open: 2000
70:Intermediate: 100
70:MGILeague-Div1: 100
70:MGILeague-Div2: 100
70:Masters: 100
70:Open: 250
71:Intermediate: 100
71:MGILeague-Div1: 100
71:MGILeague-Div2: 100
71:Masters: 100
71:Open: 200
72:Intermediate: 100
72:MGILeague-Div1: 100
72:MGILeague-Div2: 100
72:Masters: 100
72:Open: 200
73:Intermediate: 100
73:MGILeague-Div1: 100
73:MGILeague-Div2: 100
73:Masters: 100
73:Open: 200
74:Intermediate: 100
74:MGILeague-Div1: 100
74:MGILeague-Div2: 100
74:Masters: 100
74:Open: 200
75:Intermediate: 100
75:MGILeague-Div1: 100
75:MGILeague-Div2: 100
75:Masters: 100
75:Open: 200
76:Intermediate: 100
76:MGILeague-Div1: 100
76:MGILeague-Div2: 100
76:Masters: 100
76:Open: 200
77:Intermediate: 100
77:MGILeague-Div1: 100
77:MGILeague-Div2: 100
77:Masters: 100
77:Open: 200
78:Intermediate: 100
78:MGILeague-Div1: 100
78:MGILeague-Div2: 100
78:Masters: 100
78:Open: 200
79:Intermediate: 100
79:MGILeague-Div1: 100
79:MGILeague-Div2: 100
79:Masters: 100
79:Open: 200
7:Intermediate: 1250
7:MGILeague-Div1: 2750
7:MGILeague-Div2: 1250
7:Masters: 2500
7:Open: 2000
80:Intermediate: 100
80:MGILeague-Div1: 100
80:MGILeague-Div2: 100
80:Masters: 100
80:Open: 200
81:Intermediate: 100
81:MGILeague-Div1: 100
81:MGILeague-Div2: 100
81:Masters: 100
81:Open: 150
82:Intermediate: 100
82:MGILeague-Div1: 100
82:MGILeague-Div2: 100
82:Masters: 100
82:Open: 150
83:Intermediate: 100
83:MGILeague-Div1: 100
83:MGILeague-Div2: 100
83:Masters: 100
83:Open: 150
84:Intermediate: 100
84:MGILeague-Div1: 100
84:MGILeague-Div2: 100
84:Masters: 100
84:Open: 150
85:Intermediate: 100
85:MGILeague-Div1: 100
85:MGILeague-Div2: 100
85:Masters: 100
85:Open: 150
86:Intermediate: 100
86:MGILeague-Div1: 100
86:MGILeague-Div2: 100
86:Masters: 100
86:Open: 150
87:Intermediate: 100
87:MGILeague-Div1: 100
87:MGILeague-Div2: 100
87:Masters: 100
87:Open: 150
88:Intermediate: 100
88:MGILeague-Div1: 100
88:MGILeague-Div2: 100
88:Masters: 100
88:Open: 150
89:Intermediate: 100
89:MGILeague-Div1: 100
89:MGILeague-Div2: 100
89:Masters: 100
89:Open: 150
8:Intermediate: 1250
8:MGILeague-Div1: 2750
8:MGILeague-Div2: 1250
8:Masters: 2500
8:Open: 2000
90:Intermediate: 100
90:MGILeague-Div1: 100
90:MGILeague-Div2: 100
90:Masters: 100
90:Open: 150
91:Intermediate: 100
91:MGILeague-Div1: 100
91:MGILeague-Div2: 100
91:Masters: 100
91:Open: 100
92:Intermediate: 100
92:MGILeague-Div1: 100
92:MGILeague-Div2: 100
92:Masters: 100
92:Open: 100
93:Intermediate: 100
93:MGILeague-Div1: 100
93:MGILeague-Div2: 100
93:Masters: 100
93:Open: 100
94:Intermediate: 100
94:MGILeague-Div1: 100
94:MGILeague-Div2: 100
94:Masters: 100
94:Open: 100
95:Intermediate: 100
95:MGILeague-Div1: 100
95:MGILeague-Div2: 100
95:Masters: 100
95:Open: 100
96:Intermediate: 100
96:MGILeague-Div1: 100
96:MGILeague-Div2: 100
96:Masters: 100
96:Open: 100
97:Intermediate: 100
97:MGILeague-Div1: 100
97:MGILeague-Div2: 100
97:Masters: 100
97:Open: 100
98:Intermediate: 100
98:MGILeague-Div1: 100
98:MGILeague-Div2: 100
98:Masters: 100
98:Open: 100
99:Intermediate: 100
99:MGILeague-Div1: 100
99:MGILeague-Div2: 100
99:Masters: 100
99:Open: 100
9:Intermediate: 1250
9:MGILeague-Div1: 2750
9:MGILeague-Div2: 1250
9:Masters: 2500
9:Open: 2000
